extensions/keepassxc/src/index.tsx

```diff
@@ -55,33 +57,60 @@ export default function Command() {
           keywords={entry.split("/").slice(1)}
           actions={
             <ActionPanel>
-              <Action
+              <ActionPanel.Item
                 title="Paste"
-                icon={Icon.BlankDocument}
+                icon={Icon.TextDocument}
                 onAction={() => {
-                  pastePassword(entry).then(() => closeMainWindow());
+                  copyAndPastePassword(entry).then(() => closeMainWindow());
                 }}
               />
-              <Action
+              <ActionPanel.Item
                 title="Copy Password"
                 icon={Icon.Clipboard}
                 shortcut={{ modifiers: ["cmd"], key: "enter" }}
                 onAction={() => {
                   copyPassword(entry).then(() => closeMainWindow());
```

Comment:
```suggestion
                  copyPassword(entry).then(() => closeMainWindow()).catch(errorHandler);
```
I have forgot to add errorHandler in the end, it's my mistake, Fix it in this commit.